default.nix

```diff
@@ -87,7 +87,7 @@ rec {
   # libraries and documentation. We disable running the test suite on Nix
   # builds, as they require a database to be set up.
   postgrestPackage =
-    lib.dontCheck (lib.enableCabalFlag postgrest "FailOnWarn");
+    lib.dontCheck postgrest;
```

Comment:
Have no idea what I did here - tried to remove the "FailOnWarn" flag, because I had enabled it by default now. Didn't test that bit here - @monacoremo does the change make sense?

Reply:
LGTM!